app/src/main/kotlin/com/wire/android/ui/migration/MigrationViewModel.kt

```diff
@@ -116,6 +116,7 @@ class MigrationViewModel @Inject constructor(
                 is MigrationData.Result.Failure.Account.Specific -> MigrationState.Failed.Account.Specific(data.userName, data.userHandle)
                 is MigrationData.Result.Failure.Messages -> MigrationState.Failed.Messages(data.errorCode)
                 MigrationData.Result.Failure.NoNetwork -> MigrationState.Failed.NoNetwork
+                is MigrationData.Result.Failure.Unknown -> MigrationState.Failed.Unknown(data.throwable)
```

Comment:
`MigrationState.Failed.Unknown` does exactly the same as `MigrationState.Failed.Account.Any` so I would prefer not adding another type and returning only types that represent different states and actions, what do you think?